Extension/package.json

```diff
@@ -205,7 +205,7 @@
           "type": "string",
           "enum": [
             "Forward Slash",
-            "Back Slash"
+            "Backslash"
```

Comment:
What?

Reply:
Okay, looks like "Backslash" is a special word.

Reply:
nah, just a regular word 😉